### ISSUE TITLE
In OMP the Dublin core meta data is incorrect when chapters are added to a book

### ISSUE THREAD

**Issue by gemusehandler** [pkp/pkp-lib#6841]
**Describe the bug**

OMP3.3.0.2

I have previously asked to look into the Google Scholar indexing. To test if Scholar now finally picks up OMP to be indexed I have started a new repository. The aim is to let this one grow. I am still waiting for the publications to appear in Scholar.

I had in the mean time a look at the OAI-PMH interface. Dublin Core if I am correct. I notice that in the case of books with chapters you find repetitive entries of the full publication. The repetition is equal to the number of chapters. The chapters themselves are not displayed, no chapter titles, authors, or abstracts.

Here is the repository:
http://bookrxiv.ac/index.php/books

And here is the list of OAI records:
http://bookrxiv.ac/index.php/books/oai/?verb=ListRecords&metadataPrefix=oai_dc

My impression is that something is wrong here. Or?

Disclaimer: although we did some serious work to improve the visual performance of OMP, our modifications are limited to the .less files and the frontend templates.

**Comment by asmecher** [comment]
@gemusehandler, do you have a Publication Format created for each chapter, or all the chapters collected into a single Publication Format?

**Comment by gemusehandler** [comment]
Each chapter has its own publication format.

![Screenshot 2021-03-09 at 21 47 32](https://user-images.githubusercontent.com/4614219/110536221-7ca4a180-8121-11eb-81e4-09e8ab5a1f44.jpg)

**Comment by asmecher** [comment]
@gemusehandler, I'm afraid that explains the problem -- the OAI interface expects chapters to be collected within a single publication format, with the publication format representing the "physical" characteristics of the collection of chapters.

**Comment by gemusehandler** [comment]
Okay, I have changed that now with two books. The result is a number of deleted entries. The modified books appear now only once. But I see that the chapters disappear and do not return. If that is the design: okay.

**Comment by NateWr** [comment]
Going to close this for now, but feel free to reopen it if there is anything further that needs to be discussed.

**Comment by gemusehandler** [comment]
In fact there is a follow-up issue.

I have changed the content like Alec suggested. It looks fine but... I am not so sure about the data you don't see.

The DC-metadata of the chapters of an edited book is included in the header of the PDF download. It displays the author of the chapter correctly (DC.Creator.PersonalName), as well as the title of the chapter (DC.Title). However it does display the synopsis (DC.Description) of the full book and not the synopsis that was submitted together with the other metadata of the chapter.

Shouldn't it pull the synopsis of the chapter at this point from the database?

**Comment by gemusehandler** [comment]
In fact there is a follow-up issue.

I have changed the content like Alec suggested. It looks fine but... I am not so sure about the data you don't see.

The DC-metadata of the chapters of an edited book is included in the header of the PDF download. It displays the author of the chapter correctly (DC.Creator.PersonalName), as well as the title of the chapter (DC.Title). However it does display the synopsis (DC.Description) of the full book and not the synopsis that was submitted together with the other metadata of the chapter.

Shouldn't it pull the chapter’s synopsis of the chapter at this point from the database in stead of the book’s synopsis?

**Comment by NateWr** [comment]
Thanks @gemusehandler. That sounds right, but I'll let @asmecher confirm because I'm not too familiar with the DC metadata.

**Comment by asmecher** [comment]
@withanage, I think this might be a good one for you to look at!

**Comment by asmecher** [comment]
@withanage, I've merged the PR above; could you forward-port it to `main`?

**Comment by withanage** [comment]
@asmecher 

Forward-porting PR created https://github.com/pkp/omp/pull/1018   thus closing the issue.

**Comment by asmecher** [comment]
Thanks, Dulip!
